Branch-SDK/Branch-SDK/BranchActivityItemProvider.m

```diff
@@ -10,6 +10,16 @@
 #import "Branch.h"
 #import "BNCSystemObserver.h"
 
+// Define constants so that iOS 6 won't crash. Don't care about their values since you can't
+// possible access them, just necessary for the sake of not accessing missing symbols.
+#ifndef UIActivityTypeAddToReadingList
+#define UIActivityTypeAddToReadingList @""
```

Comment:
I'm using the absence of `UIActivityTypeAddToReadingList` as a marker. It's lazy, but I don't think we're going to see a scenario where just one of those is missing, since those 5 are all added with iOS 7.